django/db/models/manager.py

```diff
@@ -77,8 +77,7 @@ def __init__(self):
     def __str__(self):
         """ Return "app_label.model_label.manager_name". """
         model = self.model
-        app = model._meta.app_label
-        return '%s.%s.%s' % (app, model._meta.object_name, self.name)
+        return '%s.%s' % (model._meta.label, self.name)
```

Comment:
You can remove the `model = self.model` above and use `self.model._meta.label` here.